nerfstudio/models/splatfacto.py

```diff
@@ -787,7 +796,8 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
                 W,
                 background=torch.zeros(3, device=self.device),
             )[..., 0:1]  # type: ignore
-            depth_im = torch.where(alpha > 0, depth_im / alpha, depth_im.detach().max())
+            depth_im[alpha > 0] = depth_im[alpha > 0] / alpha[alpha > 0]
```

Comment:
looks like an unintentional merge conflict, the `torch.where` line is correct

Reply:
I will try rerun my experiment with `torch.where` line is applied

Reply:
fixed!